### DISCUSSION TITLE
Can't get ray working

### DISCUSSION THREAD

**Discussion by PerchPeekMichael** [spatie/ray#535] [Q&A]
I am attempting to get my Ray client to work but for some reason it just won’t pick up my ray() calls. 
 
I am running the project using WSL2 on Windows 11. It was working at some point but had suddenly stopped working. 
 
I have reserved my PC’s IP on the DHCP and set RAY_HOST to the ip address of my machine. It is still picking up nothing. 
 
Any recommendations on how to further debug this?

**Comment by AdrianMrn** [comment]
Hi @PerchPeekMichael, thanks for trying out Ray!

I think some of the people in these threads might have had similar issues to yours with WSL and Ray, and have shared their solutions for it:
- https://github.com/spatie/ray/issues/191
- https://github.com/spatie/ray/issues/235

**Reply by PerchPeekMichael** [reply]
It seems updating version has indeed fixed my issue :) Thanks!

**Reply by AdrianMrn** [reply]
Great to hear! Let us know if you have any suggestions :)
